.github/workflows/actions.yml

```
@@ -0,0 +1,52 @@
+name: Continuous Integration
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
+jobs:
+  specs:
+    runs-on: zendesk-stable
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby:
+          - 2.6.6 # Current Classic
+          - 2.7.1
+        gemfile:
+          - gemfiles/rails4.2.gemfile # Current Classic
+          - gemfiles/rails5.0.gemfile
+          - gemfiles/rails5.1.gemfile
+          - gemfiles/rails5.2.gemfile
+        exclude:
+          - {ruby: 2.7.1, gemfile: 'gemfiles/rails4.2.gemfile'} # NoMethodError: undefined method `new' for BigDecimal:Class
+    steps:
+      - uses: zendesk/checkout@v2
+      - uses: zendesk/setup-ruby@v1.48.0
+        with:
+          ruby-version: ${{ matrix.ruby }}
+      - name: install_gems
+        env:
+          BUNDLE_GEMFILE: ${{ matrix.gemfile }}
+        run: |
+          bundle install --jobs=3 --retry=3
+          bundle update --bundler
```

this should do the same:
```suggestion
        bundler-cache: true
```